internal/controller/githubissue_controller_test.go

```
@@ -46,13 +48,15 @@ var _ = Describe("GithubIssue Controller", func() {
-						Namespace: "default",
-					},
-					// TODO(user): Specify other spec details if needed.
-				}
+				// resource := &trainingv1alpha1.GithubIssue{
+				// 	ObjectMeta: metav1.ObjectMeta{
+				// 		Name:      resourceName,
+				// 		Namespace: "default",
+				// 	},
+				// 	// TODO(user): Specify other spec details if needed.
+				// }
+				resource := getTestCR(resourceName, "https://github.com/Shai1-Levi/githubissues-operator", 
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)